src/lib/project/compare-branches.ts

```diff
@@ -0,0 +1,31 @@
+import debug = require("debug");
+import { requestsManager } from "snyk-request-manager";
+import { updateProject } from "../api/project";
+
+export async function compareAndUpdateBranches(
+    requestManager: requestsManager,
+    snykDefaultBranch: string,
+    githubBranch: string,
+    projectId: string,
+    orgId: string,
+  ): Promise<{projectUpdated:boolean}> {
+
+    let projectUpdated = false
+
+      if (snykDefaultBranch != githubBranch) {
+        debug(`default branch update needed for project ${projectId}\n`);
+        try {
+          const project = await updateProject(requestManager, orgId, projectId, {branch: githubBranch});
+          if (project.branch) {
```

Comment:
not a blocker, but not sure why this would definitely tell us it was successful? If we got a non error response I assume it worked. This check could go